### ISSUE TITLE
I can't transfer .pb file to .tflite file

### ISSUE THREAD

**Issue by hsmnzaydn** [tensorflow/tensorflow#18289]
Hi,
I trained a model with keras framework. And i saved model as .pb file. This model takes input as "String Array" and gives out as "String Array" .I want use this model on Android device. Therefore, i use TOCO but i can't give true  --input_shaped and --inference_type parameters.  I use this toco command

bazel-bin/tensorflow/contrib/lite/toco/toco --input_file=modelimtest.pb --input_format=TENSORFLOW_GRAPHDEF --output_format=TFLITE --output_file=modelim.lite --inference_type=QUANTIZED_UINT8 --input_arrays=input_1 --output_arrays=activation_1/truediv input_shaped=

How can i change this command line for my problem?

------------------------

### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
- **TensorFlow installed from (source or binary)**:Source
- **TensorFlow version (use command below)**:1.7
- **Python version**: 3.4
- **Bazel version (if compiling from source)**:
- **GCC/Compiler version (if compiling from source)**:
- **CUDA/cuDNN version**: Cuda 9.0
- **GPU model and memory**: GTX 1050
- **Exact command to reproduce**:

You can collect some of this information using our environment capture script:

https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh

You can obtain the TensorFlow version with

python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"

### Describe the problem
Describe the problem clearly here. Be sure to convey here why it's a bug in TensorFlow or a feature request.

### Source code / logs
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached. Try to provide a reproducible test case that is the bare minimum necessary to generate the problem.

**Comment by marcomele** [comment]
same problem here, I tried and run this:
`
bazel run --config=opt tensorflow/tensorflow/contrib/lite/toco:toco -- \
> --input_file=./frozen_model/froze_output.pb \
> --output_file=./model.tflite \                      
> --inference_type=FLOAT \                            
> --input_shape=17,100,100,9 \                        
> --input_array=dense_1_input \                       
> --output_array=dense_3_1/Softmax                    
WARNING: Config values are not defined in any .rc file: opt
ERROR: Skipping 'tensorflow/tensorflow/contrib/lite/toco:toco': error loading package 'tensorflow/tensorflow/contrib/lite/toco': Extension file not found. Unable to load package for '//tensorflow/core:platform/default/build_config.bzl': BUILD file not found on package path
WARNING: Target pattern parsing failed.
ERROR: error loading package 'tensorflow/tensorflow/contrib/lite/toco': Extension file not found. Unable to load package for '//tensorflow/core:platform/default/build_config.bzl': BUILD file not found on package path
INFO: Elapsed time: 0.103s
FAILED: Build did NOT complete successfully (0 packages loaded)
    currently loading: tensorflow/tensorflow/contrib/lite/toco
ERROR: Build failed. Not running target
`
and that is what I got. Not only I'm not 100% sure that the parameters are right, but I've never used bazel and have no idea on how to proceed.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @bignamehyp: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by andrehentz** [comment]
Support for string tensors is not perfect at the moment. We are working on it.

**Comment by andrehentz** [comment]
In any case, the error you are seeing is probably related to your setup. Did you run ./configure ?

**Comment by tensorflowbutler** [comment]
Nagging Assignee @andrehentz: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.
